.github/workflows/update-canary.yml

```diff
@@ -0,0 +1,34 @@
+name: Update canary branch
+
+on:
+  schedule:
+    - cron: '0 6 * * *'
+
+jobs:
+  updateCanary:
+    if: github.repository == 'nodejs/node-v8'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone node-v8
+        uses: actions/checkout@v2
+        with:
+          path: node-v8
+
+      - name: Clone V8
+        uses: actions/checkout@v2
+        with:
+          repository: v8/v8
+          ref: lkgr
+          path: v8
+
+      - name: Install Node.js
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
+
+      - name: Install dependencies
+        run: |
+          npm install -g node-core-utils@latest
```

Comment:
Maybe install from github repo, we can have latest patch.

Reply:
I don't think that's necessary, if we need the newest version we can always make a ncu release (both myself and targos have access to do so)